lib/version.js

```diff
@@ -96,8 +96,7 @@ files.forEach(file => {
   metadata.original_id = metadata.id;
   metadata.id = 'version-' + version + '-' + metadata.id;
 
-  const targetFile =
-    CWD + '/versioned_docs/version-' + version + '/' + path.basename(file);
+  const targetFile = versionFolder + '/' + path.basename(file);
```

Comment:
`versionFolder` is defined above as:

`const versionFolder = CWD + '/versioned_docs/version-' + version;`

So might as well not repeat the same full path again.